doc/GUIDE.md

```diff
@@ -266,9 +299,13 @@ Not all types of macros are supported though. Only macros that have a primitive
 value, a string, or a pointer type are supported. Most notably, function-like macros are
 not supported by jextract. For function-like macros, alternatives include re-writing the
 code inside the macro in Java, using the FFM API, or writing a small C library which wraps
-the function-like macro in a proper exported C function, that can then be linked against
+the function-like macro in a proper exported C function that can then be linked against
 through the FFM API.
 
+Note that for macros, jextract only generates an accessor when it sees a macro definition,
+like the one in the example, in the header files it parses. When a macro is only defined
```

Comment:
```suggestion
like the one in the example, in the header files it parses. When a macro is defined
```

Reply:
`only` is relevant here. If a macro is defined using `#define` _and_ on the command line, jextract will generate an accessor.

Reply:
Ok, removed.